docker/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '3'
+
+services:
+    postgres:
+        image: postgres
+        ports:
+            - '5433:5432'
+    php:
+        build: php-fpm
+        ports:
+            - '9002:9000'
```

Comment:
PHP-FPM will only be consumed by your nginx container, so there is no need for a port mapping here either.